.github/workflows/backport-dispatch.yaml

```diff
@@ -0,0 +1,34 @@
+name: Automated Backporting via Dispatch
+
+on:
+  workflow_dispatch:
+    inputs:
+      targetBranch:
+        description: 'Release (22.0, 24.0, ...)'
+        required: true
+        type: string
+        default: "24.0"
```

Comment:
Should we have a default at all? This is likely to get quickly out of date. Also, can we provide a default if this value is already `required`?